docs/sources/operations/scalability.md

```diff
@@ -27,3 +27,63 @@ The query scheduler process itself can be started via the `-target=query-schedul
 In compute-constrained environments, garbage collection can become a significant performance factor. Frequently-run garbage collection interferes with running the application by using CPU resources. The use of memory ballast can mitigate the issue. Memory ballast allocates extra, but unused virtual memory in order to inflate the quantity of live heap space. Garbage collection is triggered by the growth of heap space usage. The inflated quantity of heap space reduces the perceived growth, so garbage collection occurs less frequently.
 
 Configure memory ballast using the ballast_bytes configuration option.
+
+## Remote Rule Evaluation
+
+This feature was introduced as part of [`LID-0002`](https://github.com/grafana/loki/pull/8129).
+
+### Background & Identification of Problem
+
+By default, the `ruler` component embeds a query engine to evaluate rules. This generally works fine, except when rules
+are complex or have to process a large amount of data regularly. Poor performance of the `ruler` manifests as recording rules metrics
+with gaps or missed alerts. This situation can be detected by alerting on the `cortex_prometheus_rule_group_iterations_missed_total` metric
+when it has a non-zero value.
+
+### Recommended Solution
+A solution to this problem is to externalize rule evaluation from the `ruler` process. The `ruler` embedded query engine
```

Comment:
Do we have a guideline whether to have a newline between heading and paragraph? I see both variants.

Reply:
I don't think it matters; the markdown renderer will sort it out - but we could add a linter @jdbaldry?